modules/roles/store.go

```diff
@@ -0,0 +1,83 @@
+package roles
+
+import (
+	"fmt"
+
+	"github.com/tendermint/basecoin"
+	"github.com/tendermint/basecoin/errors"
+	"github.com/tendermint/basecoin/state"
+	wire "github.com/tendermint/go-wire"
+)
+
+func NewPerm(role []byte) basecoin.Actor {
+	return basecoin.Actor{
+		App:     NameRole,
+		Address: role,
+	}
+}
+
+// Role - structure to hold permissioning
+type Role struct {
+	MinSigs uint32           `json:"min_sigs"`
+	Signers []basecoin.Actor `json:"signers"`
+}
```

Comment:
yes, there is... well, multiple AssumeRole statements.  One accepts 1 sig from either Jae or role:devs.  Role:devs needs 2 sigs from bucky, rigel, ethan.

Just updated code to allow multiple layers of roles.  Need to test this.